build_engine_test_images/scripts/generate_images.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+
+arch=$1
+docker_id=$2
```

Comment:
There are TF_VAR_docker_id, TF_VAR_docker_password pair and docker_id, docker_password pair in this file. Are they the same?

Reply:
TF_VAR_docker_id, TF_VAR_docker_password are redundant, I removed that